### DISCUSSION TITLE
UI bug: Navigation bar highlight persists on different pages

### DISCUSSION THREAD

**Discussion by Ashad001** [community/community#58814] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

When navigating from one page to another within GitHub, such as from the Pull Request page to the Issue page, the upper navigation bar fails to update its highlighting. Despite being on a different page, the highlight indicating the current active section remains on the Pull Request section. This UI bug causes confusion as it misrepresents the user's current location within GitHub.

Here I am on 'microsoft/STL' repo and on code page, but the bar is highlighted on pull requests:

![image](https://github.com/community/community/assets/93534298/d6de7d4b-cccc-4990-8170-d26f4074720d)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @Ashad001 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Ashad001** [reply]
Oh, thanks!
